yt_dlp/extractor/puhutv.py

```diff
@@ -191,27 +192,29 @@ def _extract_entries(self, seasons):
             if not season_id:
                 continue
             page = 1
+            per = 100
```

Comment:
if we're reverting the `_download_json` note, then we can inline this below again
```suggestion
```